core/src/main/java/com/linecorp/armeria/common/util/Exceptions.java

```diff
@@ -170,6 +173,35 @@ public static boolean isExpected(Throwable cause) {
     }
 
     /**
+     * Throws the specified exception violating the {@code throws} clause of the enclosing method.
```

Comment:
```, with an arbitrary return clause allowing this method call to terminate any non-void function. At runtime, it will never return, however, since it always throws an exception.```

might be useful - it took me a while to figure out what this method is returning.